RFC-004-revising-licenses-page.md

```diff
@@ -0,0 +1,119 @@
+- For products, they should already have licenses as part of the pages, because once downloaded or forked the license needs to be attached rather than disconnected on an external site
+- Having the license attached to the website content means readers see the license alongside the text, and don't have to wonder or ask or hunt around for licensing info
+
+### Some clarity
+
+Note that you can specify with CC BY what content is covered under the license. It isn't inherent to the license that everything in a work is covered. You always say what's covered, whether it's "`This work is licensed under…`" or "`The text of this book is licensed under…`" or "`The text of chapters 1 to 6 are licensed under…`" or "`All text aside from that appearing on pages 66-67 are licensed under…`". (All instances I've used/seen used in games.)
+
+To illustrate this point, here's text from the Creative Commons site's footer: `Except where otherwise noted, content on this site is licensed under a Creative Commons Attribution 4.0 International license.`
+
+### We can't un-CC existing content
+
+Part of the CC licenses is that we can't revoke the license or otherwise release something under a more restrictive license. Per the human-readable page, "The licensor cannot revoke these freedoms as long as you follow the license terms."
+
+By changing the licenses page, we aren't "undoing" any license terms for the content at the moment. What we're doing instead is saying all changes going forward on the site are under the new terms (including not released at all in some cases).
+
+If someone goes to a version of the site with the current license page (via github, archive.org, etc.), all of that material is still CC BY 4.0. We don't need to spell this out on the license page, but if someone inquires that's the answer.
+
+## Links
+
+* [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+* [Good Docs Project licenses page](https://thegooddocsproject.dev/licenses/)
+* [Current RFC002 file, with prior discussions about licenses](https://github.com/thegooddocsproject/request-for-comment/pull/3)
+
+## Feedback
+
+{{Reviewer name}}:
+- {{I suggest that ...}}
+    - {{Community member name}}: {{Additional comment on same topic.}}
+    - {{Proposer name}}: {{Addressed by ... / Ignored because ... / ...}}
+
+## Implementation checklist
+
+If this proposal is accepted, the following tasks must be completed:
+
+- [ ] Revise license page on website
+- [ ] Have documented our license info, procedures, assumptions, whatever we need to have documented as an org to make all of this known to all contributors
+- [ ] Merge and deploy license page revisions
+- [ ] Ensure any relevant site pages or documents has their license terms in them, and not relying on the site's license page
+- [ ] Define and address site repos' stucture needs and code/internal licencing needs based on content licensing
```

Comment:
Added this element, to make sure we cover whatever technological implications of this RFC